src/styles/block/container.scss

```diff
@@ -0,0 +1,11 @@
+.container {
+  position: relative;
+  height: 100vh;
+  display: grid;
+  flex-direction: column;
```

Comment:
The 'flex-direction: column;' property is redundant here because the container is using CSS Grid ('display: grid;'). Consider removing this line.